<!--xml-->
<launch>
  <param name="capture_date" value="9Dec2014" />
  <param name="picked_date" value="9Dec2014" />
  <param name="result_name" value="20Jan2015" />
  <param name="show_images" value="0" />
  <param name="detect_method" value="ORB" />
  <param name="filter" value="DIGITAL_FILTER" />
  
  <param name="blur_max" value="11" />
  
  <param name="order_max" value="4" />


  <param name="root_directory" value="/home/linaro/catkin_ws/src/icarus_rover_rc/" />
  <param name="template_image_name" value="Template1" />
  <node name="tune_targeting" pkg="icarus_rover_rc" type="tune_targeting" output="screen">
   
  </node>
</launch>
